pkg/handler/handler.go

```diff
@@ -13,21 +14,24 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-func New(m map[schema.GroupVersionKind]runner.Runner) sdk.Handler {
-	return &Handler{crdToPlaybook: m}
+// Interface used to handle a runner event. This will be called if a
+// mapping is found for the GVK of the event. This will be used as an override
+// for the default handle implementation. To set this for the handler, set the
+// options Handle with your implementation of this interface, and it will be
+// used.
+type Interface interface {
```

Comment:
Minor nit, but I find the name interface to be a little hard to follow, since it's already such a common term in Go code.